packages/overlay/src/OverlayTrigger.ts

```diff
@@ -217,6 +270,16 @@ export class OverlayTrigger extends LitElement {
                 if (!this.open && this.hoverContent) {
                     this.open = 'hover';
                 }
+                if (this.hasVisibleFocusInTree() && this.longpressDescriptor) {
+                    this.longpressDescriptor.innerHTML =
+                        LONGPRESS_INSTRUCTIONS.keyboard;
+                } else if (
+                    !this.hasVisibleFocusInTree() &&
+                    this.longpressDescriptor
+                ) {
+                    this.longpressDescriptor.innerHTML =
+                        LONGPRESS_INSTRUCTIONS.touch;
```

Comment:
When I run this on iOS w/ VO I get "press space and alt+down arrow for additional options" when I focus on the button - once I have done double tap and long press once then I get the appropriate touch message subsequently but the first time I get the keyboard message.

Reply:
I will look into fixing this change!